procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -5,10 +5,10 @@
    "id": "3dbf3c0c",
```

Comment:
<p>what do you mean it might take a few minutes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
It was previously referring to the NCSA ingestion. I guess I will remote it.